presto-main/src/main/java/io/prestosql/sql/analyzer/Analysis.java

```diff
@@ -850,12 +850,12 @@ public GroupingSetAnalysis(
     public static final class AccessControlInfo
     {
         private final AccessControl accessControl;
-        private final Identity identity;
+        private final Session session;
 
-        public AccessControlInfo(AccessControl accessControl, Identity identity)
+        public AccessControlInfo(AccessControl accessControl, Session session)
         {
             this.accessControl = requireNonNull(accessControl, "accessControl is null");
-            this.identity = requireNonNull(identity, "identity is null");
+            this.session = requireNonNull(session, "session is null");
         }
```

Comment:
This class doesn't need the session. It only cares about `Identity`. The caller should pass the identity instead.